src/main/scala/com/memsql/spark/MemsqlLoadDataWriter.scala

```diff
@@ -156,12 +156,12 @@ class LoadDataWriterFactory(table: TableIdentifier, conf: MemsqlOptions)
     if (loadDataFormat == MemsqlOptions.LoadDataFormat.Avro) {
       new AvroDataWriter(avroSchema, outputstream, writer, conn)
     } else {
-      new LoadDataWriter(outputstream, writer, conn)
+      new CSVDataWriter(outputstream, writer, conn)
```

Comment:
why did you rename this writer

Reply:
I think this name has more sense (we had AvroDataWriter and LoadDataWriter (which writes the CSV format))